src/cdn/settings/urls.css

```diff
@@ -1 +0,0 @@
-@import url("https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:FILL@0..1&family=Roboto:wght@300;400;500&display=swap");
```

Comment:
What about the gone `display: swap` property?

Reply:
The Roboto Font related `@import` declarations in [beer.css](https://github.com/beercss/beercss/blob/main/src/cdn/beer.css) bring in the following css files:

- `node_modules/@fontsource/roboto/400.css`
- `node_modules/@fontsource/roboto/500.css`

Both of them have `font-display: swap` for all of the `@font-face` declarations. So we are good there.

For the material symbols, the file being brought in seems to be the following:

- `node_modules/material-symbols/index.css`

This file uses `font-display: block` in its `@font-face` declarations which seems to be the default value of `font-display`. 

I believe we do not want to use `swap` for symbols because there is no fallback font for them. If `font-display: swap` is used, then while the material symbols font file loads, the end-user will see un-ligatured text.

Thoughts?

Reply:
I’m not the repo owner. I cannot tell if these import changes go into the direction decided for beercss. So I will let @leonardorafael answer :wink: